assets/panel/debugger.properties

```diff
@@ -227,28 +227,28 @@ searchPanelVariable=Filter variables (%S)
 # LOCALIZATION NOTE (breakpointMenuItem): The text for all the elements that
 # are displayed in the breakpoints menu item popup.
 breakpointMenuItem.setConditional=Configure conditional breakpoint
-breakpointMenuItem.enableSelf=Enable breakpoint
-breakpointMenuItem.enableSelf.accesskey=E
-breakpointMenuItem.disableSelf=Disable breakpoint
-breakpointMenuItem.disableSelf.accesskey=D
-breakpointMenuItem.deleteSelf=Remove breakpoint
-breakpointMenuItem.deleteSelf.accesskey=R
-breakpointMenuItem.enableOthers=Enable others
-breakpointMenuItem.enableOthers.accesskey=o
-breakpointMenuItem.disableOthers=Disable others
-breakpointMenuItem.disableOthers.accesskey=s
-breakpointMenuItem.deleteOthers=Remove others
-breakpointMenuItem.deleteOthers.accesskey=h
-breakpointMenuItem.enableAll=Enable all breakpoints
-breakpointMenuItem.enableAll.accesskey=b
-breakpointMenuItem.disableAll=Disable all breakpoints
-breakpointMenuItem.disableAll.accesskey=k
-breakpointMenuItem.deleteAll=Remove all breakpoints
-breakpointMenuItem.deleteAll.accesskey=a
-breakpointMenuItem.removeCondition.label=Remove breakpoint condition
-breakpointMenuItem.removeCondition.accesskey=c
-breakpointMenuItem.editCondition.label=Edit breakpoint condition
-breakpointMenuItem.editCondition.accesskey=n
+breakpointMenuItem.enableSelf2=Enable
+breakpointMenuItem.enableSelf2.accesskey=E
+breakpointMenuItem.disableSelf2=Disable
+breakpointMenuItem.disableSelf2.accesskey=D
+breakpointMenuItem.deleteSelf2=Remove
+breakpointMenuItem.deleteSelf2.accesskey=R
+breakpointMenuItem.enableOthers2=Enable others
+breakpointMenuItem.enableOthers2.accesskey=o
+breakpointMenuItem.disableOthers2=Disable others
+breakpointMenuItem.disableOthers2.accesskey=s
+breakpointMenuItem.deleteOthers2=Remove others
+breakpointMenuItem.deleteOthers2.accesskey=h
+breakpointMenuItem.enableAll2=Enable all
+breakpointMenuItem.enableAll2.accesskey=b
+breakpointMenuItem.disableAll2=Disable all
+breakpointMenuItem.disableAll2.accesskey=k
```

Comment:
no other letter available?

Reply:
there are a lot of other items in the context menu so it is tough to come up w/ one